app/serializers/remote_search_form_serializer.rb

```diff
@@ -0,0 +1,31 @@
+class RemoteSearchFormSerializer < ActiveModel::Serializer
+  self.root = false
+
+  attributes :sort, :query
+
+  def sort
+    'registered_name'
+  end
+
+  def query
+    {
+      'filtered' => {
+        'filter' => {
+          'in' => {
+            'other_advice_methods' => remote_advice_method_ids
+          }
+        }
+      }
+    }
+  end
+
+  private
+
+  def remote_advice_method_ids
```

Comment:
This approach feels very fragile. If the text of the advice method changes, that should not require changes here too. We should be matching these up via identity rather than value.